src/app/pages/private/collection/collection.page.ts

```diff
@@ -181,4 +194,20 @@ export default class CollectionPage implements OnInit {
   getCollectedCardsForRarity(cardsForCurrentRarity: Card[]): Card[] {
     return cardsForCurrentRarity.filter((card: Card) => card.status === CardStatus.Collected);
   }
+
+  getCanMarkAllAsCollected() {
+    return (
+      !this.getIsCardsListEmpty(this.cardsForCurrentRarity) && this.getIsSomeCardUncollected(this.cardsForCurrentRarity)
+    );
+  }
+
+  getIsCardsListEmpty(cards: Card[] | null): boolean {
+    return cards === null || cards.length === 0;
+  }
+
+  getIsSomeCardUncollected(cards: Card[] | null) {
```

Comment:
don't forget to add returning types for methods

Reply:
my bad, added return type for method